tools/tock-register-interface/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+//! Tock Register Interface
+//!
+//!
+
+#![feature(const_fn)]
+#![no_std]
+
+pub mod regs;
+
+#[macro_use]
+pub mod macros;
```

Comment:
So the only thing I would want it the export the structs directly here by:

```rust
pub use regs::{ReadOny, WriteOnly, LocalRegisterCopy, Field, FieldValue}
```
Then if I use this crate it's enough to 
```rust
extern crate tock_regs;
use tock_regs::ReadWrite;
```

Instead of 
```rust
extern crate tock_regs;
use tock_regs::regs::ReadWrite;
```

Reply:
That seems like a good suggestion. I will do that.